marklogic-data-hub-central/ui/src/components/header/system-info.tsx

```diff
@@ -172,7 +172,7 @@ const SystemInfo = (props) => {
               <Col>
                 <Card size="small" className={styles.download} >
                   <div className={styles.title}>Download Project Files</div>
-                  <p>Download a zip containing files for everything that Hub Central supports creating and updating and all Data Hub Framework project files. Can be used to deploy the project in a local environment or add the project to a version control system.</p>
+                  <p>Download a zip containing a complete set of files such that a project can be deployed locally or checked into version control. In addition to the files Hub Central supports creating and updating, the zip contains all Data Hub project files.</p>
```

Comment:
Similarly, this could be
`
In addition to the files that can be created and updated via Hub Central, the zip also contains all Data Hub project files as well.
`